### ISSUE TITLE
Multi-account support: Messages screen

### ISSUE THREAD

**Issue by giomfo** [vector-im/vector-android#576]
We would like a proposal for handling multiple accounts display in Messages screen.
- How to display the favorite rooms?
- Shall we interleave or separate rooms joined by different accounts?
- How to handle rooms joined by several logged in accounts?

see https://github.com/vector-im/vector-ios/issues/589

**Comment by antikewl** [comment]
Profile switcher menu
https://zpl.io/ZmeIfM
https://zpl.io/CMJiJ

**Comment by thekyriarchy** [comment]
@antikewl are there links that don't require an account?

**Comment by ara4n** [comment]
Here are the proposed designs, which are pretty cute suggestion of how to rapidly switch account (although I'm not sure how well it works in practice, given the alt account may not even be in that room!)

<img width="367" alt="screen shot 2017-10-29 at 21 24 13" src="https://user-images.githubusercontent.com/1294269/32148489-9dc1700e-bcef-11e7-8849-e65b8a0ea146.png">
<img width="367" alt="screen shot 2017-10-29 at 21 23 54" src="https://user-images.githubusercontent.com/1294269/32148490-9de3dbee-bcef-11e7-8ff8-d444b24ed070.png">

**Comment by alexgleason** [comment]
![screenshot_20180613-205237](https://user-images.githubusercontent.com/3639540/41385791-c34751ea-6f4b-11e8-8387-e4e547b9d354.png)

I think it should work like this. You select your account from the drawer on the left, which already exists, and it changes your entire view of rooms etc.

**Comment by r4dh4l** [comment]
What about another horizontal bar at the bottom of the home screen with Icons for the account avatars to switch the profile?
In riot desktop/web this could be another vertical bar left of the community bar to switch accounts.

**Comment by PrivSecurity** [comment]
> 
> 
> ![screenshot_20180613-205237](https://user-images.githubusercontent.com/3639540/41385791-c34751ea-6f4b-11e8-8387-e4e547b9d354.png)
> 
> I think it should work like this. You select your account from the drawer on the left, which already exists, and it changes your entire view of rooms etc.

I agree and that is the way it works in most other apps with multi-account support, e.g. Telegram Private Messenger, Jami.net (former Ring), surespot, Reddit, Twitter, Google Hangouts. That way the accounts are clearly separated from each other. Reddit might be especially comparable since they have both forums, chats, direct messages and so on and I think it works very well separating the accounts that way.

**Comment by Powersource** [comment]
Any news on this?

And I agree that it should work like in the screenshot in the previous comment.
